examples/Ep_spacing.py

```diff
@@ -0,0 +1,7 @@
+from pyspecdata import *
+import SpinCore_pp
+#config_dict = SpinCore_pp.configuration("active.ini")
+with figlist_var() as fl:
+    Ep_powers = gen_Ep_powers(sim_phalf = 0.2, sim_Emax = 10.0, sim_pmax = 3.16,
```

Comment:
how is this supposed to work?  where is the function by this name?
also, you are not pulling from the config dict, which was the goal

Reply:
as discussed in person, this needs to use active.ini

Reply:
actually, changes to the config dict are going to be required, as well